DatadogTrace/Sources/OpenTracing/OTTracer.swift

```diff
@@ -34,7 +35,8 @@ public protocol OTTracer {
     func startRootSpan(
         operationName: String,
         tags: [String: Encodable]?,
-        startTime: Date?
+        startTime: Date?,
+        customSamplingRate: SampleRate?
```

Comment:
As per our [conventions](https://datadoghq.atlassian.net/wiki/spaces/RUMP/pages/3027638506/Public+API+Convention+in+iOS+SDK+2.x#Rule-7%3A-Use-of-standard-Datadog-and-RUM-terminology), `customSamplingRate` is not coherent with other APIs. It should be `customSampleRate`

Also, this parameter must be explained in inline documentation - what it does and what sampling does it override.